docs/project/get-help.md

```diff
@@ -16,7 +16,7 @@ parent = "mn_opensource"
 }
 </style>
 
-# Where to chat or get help
+# Where to Chat or Get Help
```

Comment:
Oh! Actually, we use sentence capitalization; see the docs style guide; https://docs.docker.com/project/doc-style/#capitalization. Could you change this back? :+1: